openwisp_users/tests/test_admin.py

```diff
@@ -317,7 +317,7 @@ def test_operator_change_organization(self):
                 f'admin:{self.app_label}_organization_change', args=[default_org.pk]
             )
         )
-        self.assertContains(
+        self.assertNotContains(
```

Comment:
I don't understand why this test is being changed.
Was the test wrong? Or has the behavior changed?
Can you explain please?

Reply:
Yes it was, the operator is not an admin of default-org nor a superuser so he shouldn't be able to edit it.